ejb/WFLY-11622_Multiple_delivery_group_for_MDBs.adoc

```diff
@@ -0,0 +1,65 @@
+= WFLY-11622 Multiple Delivery Group Support for Message Driven Beans
+:author:            Flavia Rainone
+:email:             frainone@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Currently an MDB can belong to a delivery-group, a group identified by a String name.
+If the MDB belongs to a delivery-group, delivery of messages for that MDB will be enabled only when the delivery-group is active.
+If the MDB belongs to a delivery-group and is clustered singleton, delivery wlil be enabled only when the delivery-group is active in the singleton node of the cluster.
+This REF will enhance delivery-group capability, by making possible to associate an MDB to more than one delivery-groups at once.
+Delivery will be enabled only when all delivery-groups the MDB belongs to are active. In the clustered singleton case, that implies that delivery is active only 
+in the singleton node of the cluster and only if all delivery-groups associated with the MDB are active.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11622
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-720
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:mnovak@redhat.com[Miroslav Novak]
+
+=== Affected Projects or Components
+
+Affects Wildfly EJB and requires a new EJB3 Ext API version
+
+=== Other Interested Projects
+
+EJB3 Ext API should be upgraded
+
+== Requirements
+
+=== Hard Requirements
+
+Allow more than one delivery group per MDB. If an MDB belongs to more than one delivery group, delivery is enabled if and only if all delivery groups are active.
+In clustered scenarios, delivery is active only in the singleton node of the cluster and only if all delivery groups are active for that MDB.
+Delivery groups are defined by:
+- xml, so we will need to have a new version of delivery-active schema (delivery-active:1.2)
```

Comment:
@mnovak1 we have an example at quickstart.... I'll find it and it to the doc, and I'll also write a small code sample showing how it will look like with multiple delivery groups.

Reply:
@fl4via  Thanks, I'm mostly interested how multiple delivery groups will be configured in MDB (via xml/annotation) as this will allow me to write the tests without actually having your new code.

Reply:
@mnovak1  done, please let me know if you have all the information needed now

Reply:
Thanks, looks ok to me.